cob_hand_bridge/package.xml

```diff
@@ -4,9 +4,11 @@
   <version>0.6.1</version>
   <description>The cob_hand_bridge package</description>
 
-  <maintainer email="mathias.luedtke@ipa.fraunhofer.de">Mathias Lüdtke</maintainer>
+  <maintainer email="fmw@ipa.fhg.de">Florian Weisshardt</maintainer>
+  <maintainer email="nhg@ipa.fhg.de">Nadia Hammoudeh Garcia</maintainer>
```

Comment:
I added you in consistency with the meta-package...
Are you ok with leaving this for now and discuss future maintenance of the repo in conjunction with ipa320/care-o-bot#46